source/utilities.tex

```diff
@@ -75,6 +75,25 @@
   // \ref{declval}, declval
   template<class T>
     add_rvalue_reference_t<T> declval() noexcept;   // as unevaluated operand
+
+  // \ref{utility.safeintcmp}, Safe integral comparisons
```

Comment:
```suggestion
  // \ref{utility.safeintcmp}, safe integral comparisons
```

Reply:
We should not use "safe".

Reply:
And we should not use "integral" either.

I suggest "integer comparison functions" instead.

Reply:
changed to "integer comparison functions"